src/views/htmlcontent/src/html/messagescontextmenu.component.html

```diff
@@ -0,0 +1,5 @@
+<ul class="contextMenu" style="position:absolute" [class.hidden]="!visible" [style.top]="position.y" [style.left]="position.x">
```

Comment:
This file is also for reference only, in order to be consistent with our other modules.